cloud/openstack/os_endpoint.py

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Hewlett-Packard Development Company, L.P.
+# Author: Davide Guerri <davide.guerri@hp.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+try:
+    import shade
+
+    HAS_SHADE = True
+except ImportError:
+    HAS_SHADE = False
+
+DOCUMENTATION = '''
+---
+module: keystone_endpoint
+version_added: "1.9"
+short_description: Manage OpenStack Identity (keystone) endpoints
+description:
+   - Manage endpoints from OpenStack.
+options:
+   login_user:
+     description:
+        - login username to authenticate to keystone
+     required: false
+     default: None
+   login_password:
+     description:
+        - Password of login user
+     required: false
+     default: None
+   login_tenant_name:
+     description:
+        - The tenant login_user belongs to
+     required: false
+     default: None
+   service_name:
+     description:
+        - OpenStack service name (e.g. keystone)
+     required: true
+   region:
+     description:
+        - OpenStack region to which endpoint will be added
+     required: false
+     default: 'RegionOne'
```

Comment:
This is not a useful default - I'd remove it and set the default to None instead.